src/components/masternodes/MasternodeForm.jsx

```diff
@@ -49,13 +49,14 @@ function MasternodeForm() {
    * @function
    */
   const generateSignCommand = (collateralAddress, signMessage) => {
+    // check for new addresses to validate
     if (
       collateralAddress.startsWith("sys") ||
       collateralAddress.startsWith("tsys")
     ) {
       return `signmessagebech32 ${collateralAddress} ${signMessage}`.trim();
     } else {
-      return `signmessage ${collateralAddress} ${signMessage}`.trim();
+      return `signmessagebech32 ${collateralAddress} ${signMessage}`.trim();
```

Comment:
pls refactor, why did you just rename the function without looking at the context, your two statements are identical

Reply:
Done!